src/web/pages/base.ts

```diff
@@ -83,7 +83,7 @@ class Page {
     return this.getElement(elementName).isPresent();
   }
 
-  public getNumberOfElements(elementName: string) {
+  public getNumberOfElements(elementName: any) {
```

Comment:
You union as a type, probably string | protractor.ElementArrayFinder

Reply:
Done